loader/extends.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/compose-spec/compose-go/v2/types"
 )
 
+var exclusions = []string{"extends", "depends_on", "volumes_from"}
```

Comment:
Maybe it'll be better to add a comment with an explanation of why we exclude these properties, link to the doc https://docs.docker.com/compose/how-tos/multiple-compose-files/extends/#exceptions-and-limitations, or short explanation. I think it'll help in the future to understand what's going on)